### PR TITLE
[ZF3] move NamingStrategy to inside Stategy

### PR DESCRIPTION
**Pull request by samsonasik**
implements #14

### CODE REVIEW

**Comment by samsonasik** [comment]
added updated docs PR for it :  https://github.com/zendframework/zf2-documentation/pull/1502

File: src/Hydrator/Strategy/NamingStrategy/NamingStrategyEnabledInterface.php [hunk 1]
```diff
@@ -7,9 +7,9 @@
  * @license   http://framework.zend.com/license/new-bsd New BSD License
  */
 
-namespace Zend\Stdlib\Hydrator;
+namespace Zend\Stdlib\Hydrator\Strategy\NamingStrategy;
 
-use Zend\Stdlib\Hydrator\NamingStrategy\NamingStrategyInterface;
+use NamingStrategyInterface;
```
Comment (Maks3w) [inline src/Hydrator/Strategy/NamingStrategy/NamingStrategyEnabledInterface.php]: wrong replacement

Comment (samsonasik) [inline src/Hydrator/Strategy/NamingStrategy/NamingStrategyEnabledInterface.php]: no, NamingStrategyInterface already moved to same directory, or we can just remove the use statement

**Comment by Maks3w** [comment]
Per discussion https://github.com/zendframework/zend-stdlib/issues/14

**Pull request closed without merging**